README.md

```diff
@@ -798,12 +798,12 @@ swell.payment.createElements({
     cardCvc: {
       elementId: '#card-expiry-id', // default: #cardCvc-element
     },
-    onSuccess: (result) => {
-      // optional, called on card payment success
-    },
-    onError: (error) => {
-      // optional, called on card payment error
-    },
+  },
+  onSuccess: () => {
```

Comment:
@vstudenichnik-insoft I think it should actually be inside the `card`/`paypal` etc objects because you might want different callbacks and of course would have different result object for each type

Reply:
@ericingram I think these are common callbacks and it is suitable to have it here. Besides we have separate elements option for Stripe and it’s not very convenient to have common callbacks for each element (does not work now because we call the callbacks only from the `card` element). This is even better, since we can define specific callbacks for individual elements. I think this is a good point.

Reply:
@vstudenichnik-insoft I'm not 100% clear on your point, but what if we have it in both places? If you want a single callback for success/error on any payment method, and alternately if you want separate ones for each type? Like this:

```
{ card: { onSuccess }, onSuccess, ... }
```

Either that, or we need to return a `method` to the callback so the user can determine which type it is, like this:

```
{ card { ... }, onSuccess(type, params) }
```

Reply:
@ericingram I looked at the current code a bit, well, you're right. Pull request updated